demo/restore_wallet.cc

```diff
@@ -0,0 +1,18 @@
+#include "include/defi-wallet-core-cpp/src/lib.rs.h"
+#include "include/rust/cxx.h"
+#include <iostream>
+using namespace org::defi_wallet_core;
+
```

Comment:
using namespace rust;?

Reply:
I was also considering to use rust namespace or not. There maybe some reasons not using namespace rust in the examples:
1. Since the examples are very shot, only a few lines. We don't need to bring all rust types into scope.
2. Both rust and std has string, Vec etc. To make them easier to recognise and distinguish, using full path is better, especially in the small examples. In additional, we could mention `using namespace rust` in the documents, so that users know they have an easy way to bring all rust types into scope.